src/plugins/controls/public/control_types/range_slider/range_slider.component.tsx

```diff
@@ -45,16 +45,19 @@ export const RangeSliderComponent: FC<Props> = ({ componentStateSubject }) => {
     componentStateSubject.getValue()
   );
 
-  const { value = ['', ''], id, title } = useEmbeddableSelector((state) => state);
+  const { value, id, title } = useEmbeddableSelector((state) => state);
```

Comment:
When it was previously `const { value = ['', ''], id, title }`, this would cause infinite recursion because of the new `useEffect` that depends on the value changing - therefore, to avoid this, I simply do this check as part of the new `useEffect` where I set the selected value.